crates/wdl-engine/src/value.rs

```diff
@@ -2938,7 +2975,9 @@ impl TaskPreEvaluationValue {
             n if n == TASK_FIELD_META => Some(self.meta.clone().into()),
```

Comment:
In line with your other improvements, I see these can also be simplified by removing the `n if n ==`.